packages/utils/src/provider_utils.ts

```diff
@@ -74,7 +74,7 @@ export const providerUtils = {
         } else if ((supportedProvider as any).send !== undefined) {
             // HACK(fabio): Detect if the `send` method has the old interface `send(payload, cb)` such
             // as in versions < Web3.js@1.0.0-beta.37. If so, do a simple re-mapping
-            if (_.includes((supportedProvider as any).send.toString(), 'function (payload, callback)')) {
+            if (_.includes((supportedProvider as any).send.toString().replace(" ", ""), 'function(payload,callback)')) {
```

Comment:
```suggestion
            if (_.includes((supportedProvider as any).send.toString().replace(' ', ''), 'function(payload,callback)')) {
```

Should fix the linter issue. Would you be able to update the `CHANGELOG.json` file under `packages/utils/CHANGELOG.json` to reflect this change? You can follow the previous entries (just remove the timestamp).